src/main/java/org/ow2/proactive/catalog/rest/controller/BucketController.java

```diff
@@ -154,14 +157,14 @@ public void cleanEmpty() {
                             @ApiResponse(code = 401, message = "User not authenticated"),
                             @ApiResponse(code = 403, message = "Permission denied"), })
     @RequestMapping(value = "/{bucketName}", method = DELETE)
-    @ResponseStatus(HttpStatus.FORBIDDEN)
-    public ResponseEntity<?> delete(
+    @ResponseStatus(HttpStatus.OK)
+    public BucketMetadata delete(
             @ApiParam(value = "sessionID", required = false) @RequestHeader(value = "sessionID", required = false) String sessionId,
             @PathVariable String bucketName) throws NotAuthenticatedException, AccessDeniedException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)